### PR TITLE
[morpheus/token] vm limit cpu & disk usage

### LINKED ISSUE #312: Enable [morpheus/token]vm running with less nodes/resources
Currently, when running `./scripts/run.sh` script we start a cluster with 5 nodes by default. Testing locally uses a lot a computer resources. So random `context deadline exceeded` errors appear.
It would be nice to optionally run the script passing an argument to either run less nodes or limiting resources usage.

### PR DESCRIPTION
**Pull request by najeal**
Fixes #312 

It looks weird to use `100000` CPUs. I also deleted disk value to use the default one.

### CODE REVIEW

**Comment by patrick-ogrady** [comment]
I agree that we should make this configurable (use args when running the program from the run script).

You should avoid modifying the rate limit configs (those don't actually use/preallocate any CPU). Rather, they just prevent the rate limiter from being active (not needed on a local network and can cause instability during stress tests).

**Comment by najeal** [comment]
Configuring with less node generates error woth not enough amount staked. The only stuff related to the consensus I have seen for changing required staked amount is k and alpha parameters.

**Comment by patrick-ogrady** [comment]
> Configuring with less node generates error woth not enough amount staked. The only stuff related to the consensus I have seen for changing required staked amount is k and alpha parameters.

I'm just referring to making the hardcoded values I have in the e2e tests (https://github.com/ava-labs/hypersdk/blob/d6927f297d3c0619e09154518c3416610f0e2e28/examples/tokenvm/tests/e2e/e2e_test.go#L235) a variable (https://github.com/ava-labs/hypersdk/blob/d6927f297d3c0619e09154518c3416610f0e2e28/examples/tokenvm/tests/e2e/e2e_test.go#L141).

**Comment by najeal** [comment]
This configures the subnets, but I would like to configure also the `anrCli.Start()` with a `runner_sdk.WithNumNodes()`.

**Comment by patrick-ogrady** [comment]
> This configures the subnets, but I would like to configure also the `anrCli.Start()` with a `runner_sdk.WithNumNodes()`.

I wouldn't worry about that for now. They aren't really doing anything in the background here and should have very minimal CPU/RAM usage.

If you still want to change that to "1" or something, you still wouldn't need to modify any consensus params.

**Comment by najeal** [comment]
@patrick-ogrady used vars as you suggested.

File: examples/tokenvm/tests/e2e/e2e_test.go [hunk 1]
```diff
@@ -142,6 +144,13 @@ func init() {
 		"test",
 		"'test' to shut down cluster after tests, 'run' to skip tests and only run without shutdown",
 	)
+
+	flag.UintVar(
+		&numValidators,
+		"num-validators",
+		5,
+		"number of validators by blockchain",
```
Comment (patrick-ogrady) [inline examples/tokenvm/tests/e2e/e2e_test.go:152]: nit: `per`

File: examples/morpheusvm/tests/e2e/e2e_test.go [hunk 2]
```diff
@@ -140,6 +142,13 @@ func init() {
 		"test",
 		"'test' to shut down cluster after tests, 'run' to skip tests and only run without shutdown",
 	)
+
+	flag.UintVar(
+		&numValidators,
+		"num-validators",
+		5,
+		"number of validators by blockchain",
```
Comment (patrick-ogrady) [inline examples/morpheusvm/tests/e2e/e2e_test.go:150]: nit: `per`

File: examples/morpheusvm/tests/e2e/e2e_test.go [hunk 3]
```diff
@@ -226,7 +235,7 @@ var _ = ginkgo.BeforeSuite(func() {
 
 	// Name 5 new validators (which should have BLS key registered)
 	subnet := []string{}
-	for i := 1; i <= 5; i++ {
+	for i := 1; i <= int(numValidators); i++ {
```
Comment (patrick-ogrady) [inline examples/morpheusvm/tests/e2e/e2e_test.go:238]: nit: error if 0

File: examples/tokenvm/tests/e2e/e2e_test.go [hunk 4]
```diff
@@ -221,9 +230,9 @@ var _ = ginkgo.BeforeSuite(func() {
 	// Name 10 new validators (which should have BLS key registered)
 	subnetA := []string{}
 	subnetB := []string{}
-	for i := 1; i <= 10; i++ {
+	for i := 1; i <= int(numValidators)*2; i++ {
```
Comment (patrick-ogrady) [inline examples/tokenvm/tests/e2e/e2e_test.go:233]: nit: error if 0

**Comment by patrick-ogrady** [comment]
> @patrick-ogrady used vars as you suggested.

Few nits but otherwise looks great

File: examples/morpheusvm/tests/e2e/e2e_test.go (hunk 3 above)
Comment (najeal) [reply examples/morpheusvm/tests/e2e/e2e_test.go:239]: checking input is > 0

File: examples/morpheusvm/tests/e2e/e2e_test.go [hunk 5]
```diff
@@ -165,6 +165,7 @@ var _ = ginkgo.BeforeSuite(func() {
 		gomega.Equal(modeFullTest),
 		gomega.Equal(modeRun),
 	))
+	gomega.Expect(numValidators).Should(gomega.BeNumerically(">", 0))
```
Comment (patrick-ogrady) [inline examples/morpheusvm/tests/e2e/e2e_test.go:168]: Even better. Nice!

**Pull request merged**